comp/swerve100/src/main/java/org/team100/frc2024/FieldConstants.java

```diff
@@ -4,12 +4,27 @@
 
 package org.team100.frc2024;
 
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.DriverStation.Alliance;
+
 /** Add your docs here. */
 public class FieldConstants {
+    public double SHOOTER_LEFT_SIDE_Y = 0 ;
+    public double SHOOTER_RIGHT_SIDE_Y = 0; //
+    public double SHOOTER_CENTER_Y = 0;
+    public double SHOOTER_CENTER_X = 0;
```

Comment:
whoops, I should have noticed this earlier.  in java you can only override methods, not fields.  make FieldConstants an abstract class with abstract methods like getShooterCenterX() and then implement those methods in the subclasses.